14-draft.json

```diff
@@ -975,6 +975,69 @@
               "value"
             ]
           }
+        },
+        "network_traffic": {
+          "description": "The current network traffic, in bits/second or packets/second (or both)",
+          "type": "array",
+          "items": {
+            "type": "object",
+            "properties": {
+              "properties": {
```

Comment:
@dbrgn as already mentioned, I designed the schema to match the schema of the `wind` sensor, which afaict is the only one with multiple types of measurement per sensor. If we choose a different approach here, shouldn't we also change the wind sensor for consistency's sake? This would introduce a breaking change though. (Though I agree that `values` would be a better name than `properties`.)

Reply:
Ah, you're right. Then breaking compatibility is probably not worth the rename.

@rnestler @gidsi what do you think?

Reply:
Had the same thing, would like to refactor both of them on a later date.
Right now i would also say that equality is more important and would merge it like this.